cargo-shuttle/src/args.rs

```diff
@@ -392,6 +386,28 @@ impl InitTemplateArg {
     }
 }
 
+#[derive(Parser, Clone, Debug, Default)]
+pub struct LogsArgs {
+    /// Deployment ID to get logs for. Defaults to currently running deployment
+    pub id: Option<Uuid>,
+    #[arg(short, long)]
+    /// View logs from the most recent deployment (which is not always the latest running one)
+    pub latest: bool,
+    #[arg(short, long)]
+    /// Follow log output
+    pub follow: bool,
+    /// Log format
+    #[arg(short = 'F', long)]
+    pub format: Option<LogFormat>,
+}
+
+#[derive(ValueEnum, Clone, Debug, strum::Display, strum::EnumIter)]
+#[strum(serialize_all = "kebab-case")]
+pub enum LogFormat {
+    /// Don't display timestamps and log origin tags
+    Raw,
+}
```

Comment:
I defined an enum `LogFormat` to allow logs commands to be output in various formats in the future. (This is why we call it `--raw` -> `--format raw`). If this change is not good, undo it.

Reply:
Since `raw` is the only supported format, and the other commands take `--raw`, we could keep this as `--raw` for consistency.

Reply:
I have standardized on `--raw` to be consistent with other commands. Thanks for your comments!

[f24c072](https://github.com/shuttle-hq/shuttle/pull/1653/commits/f24c0721694214fd7cc3c1fbc785c22ea7f278a5)